src/Screens/ChatScreen.tsx

```diff
@@ -189,74 +193,58 @@ export default function ChatScreen({route, navigation}: any) {
     const message = {
       dialogId: dialog?.id,
       body: pendingMessages[0].text,
+      properties: {
+        type: 'text',
+      },
       saveToHistory: true,
     };
 
-    QB.chat
-      .sendMessage(message)
-      .then(function () {
-        /* send successfully */
-      })
-      .catch(function (e) {
-        /* handle error */
-      });
+    QB.chat.sendMessage(message);
     sendPushNotification(friend?.deviceToken, {
       title: user?.fullName,
       body: pendingMessages[0].text,
     });
   }
 
-  async function handleLoadEarlier() {
-    if (!messagePaginator.hasNextPage) {
-      setLoadEarlier(false);
-      return;
-    }
-    setIsLoadingEarlier(true);
-    const nextPaginator = await messagePaginator.nextPage();
-    setMessagePaginator(nextPaginator);
-    setMessages(currentMessages =>
-      GiftedChat.prepend(currentMessages, nextPaginator.items.map(mapMessage)),
-    );
-    setIsLoadingEarlier(false);
-  }
-
-  async function handleSendImage(params: any) {
-    const result: any = await launchImageLibrary({mediaType: 'photo'});
+  async function handleSendFile(type: any) {
+    const result: any = await launchImageLibrary({mediaType: type});
     if (!result?.didCancel) {
-      let name = JSON.stringify(new Date().getTime());
-      const reference = storage().ref(`images/${name}`);
-      const task = reference.putFile(result?.assets[0]?.uri);
-      task.on('state_changed', taskSnapshot => {
-        console.log(
-          `${taskSnapshot.bytesTransferred} transferred out of ${taskSnapshot.totalBytes}`,
-        );
-      });
-      task.then(async () => {
-        console.log('Image uploaded to the bucket!');
-        await storage()
-          .ref(`images/${name}`)
-          .getDownloadURL()
-          .then(async url => {
-            await chatkitty.sendMessage({
-              channel: channel,
-              file: {
-                name: result?.assets[0]?.fileName,
-                size: result?.assets[0]?.fileSize,
-                contentType: result?.assets[0]?.type,
-                url: url,
-              },
-              progressListener: {
-                onStarted: () => {},
-                onProgress: progress => {},
-                onCompleted: result => {},
-              },
-            });
-            // sendPushNotification(participantDetails?.properties?.deviceToken, {
-            //   title: user?.displayName,
-            //   body: 'image',
-            // });
+      setSending(true);
+      const contentUploadParams = {
+        url: result?.assets[0]?.uri, // path to file in local file system
+        public: false,
+      };
+      QB.content
+        .upload(contentUploadParams)
+        .then(async (file: any) => {
+          const contentGetFileUrlParams = {uid: file.uid};
```

Comment:
Please Use ES6+ Features. Modern JavaScript features can make your code more concise and easier to read. For example, you can use  destructuring.
const { uid, contentType } = file;
const contentGetFileUrlParams = { uid };